app/controllers/conversations.js

```diff
@@ -96,6 +104,17 @@ class ConversationsController extends BaseController {
       await participant.save();
     }
 
+    await this.conversationRepository.showConversation(
+      ws,
+      requestId,
+      conversationObj.visibleParams(),
+      participants,
+      {
```

Comment:
```
{
  title: "New conversation create",
  body: "<userfullname | userlogin> created a new conversation"
}
```

and we can move push notification body formation inside 

 `notifyAboutConversationCreate` and `notifyAboutConversationUpdate`